static/src/javascripts/lib/config.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import pad from 'lib/pad';
+// eslint-disable-next-line guardian-frontend/global-config
+const config = window.guardian.config;
```

Comment:
Hmm, you mean where `Object.assign(config, allTheNewMethodsHere)` happens, make sure that we're not overriding properties from config?